lua/ide/components/bufferlist/component.lua

```diff
@@ -154,9 +172,18 @@ BufferListComponent.new = function(name, config)
                 name = libbuf.get_unique_filename(vim.api.nvim_buf_get_name(buf)),
                 icon = icon,
                 id = buf,
+                is_current = (cur_buf == buf)
             }
         end, libbuf.get_listed_bufs())
         self.bufs = bufs
+        if self.config.current_buffer_top then
+            table.sort(self.bufs, function(a,_)
```

Comment:
You should use the `sort.lua` module that was added, since `table.sort` is an unstable sorting algorithm, whereas `sort.lua` uses a stable algorithm. The function signature is the same.

By stable I mean, items that do not need to be moved are guaranteed _not_ to be moved: https://en.wikipedia.org/wiki/Sorting_algorithm#Stability